airflow/settings.py

```diff
@@ -605,3 +605,7 @@ def initialize():
 AIRFLOW_MOVED_TABLE_PREFIX = "_airflow_moved"
 
 DAEMON_UMASK: str = conf.get("core", "daemon_umask", fallback="0o077")
+
+# AIP-52: setup/teardown (experimental)
+# This feature is not complete yet, so we disable it by default.
+_ENABLE_AIP_52 = bool(os.environ.get("AIRFLOW_ENABLE_AIP_52", ""))
```

Comment:
```suggestion
_ENABLE_AIP_52 = bool(os.environ.get("AIRFLOW_ENABLE_AIP_52", "false").lower() in ("true", "t", "yes", "y"))
```

Reply:
Setting "" as the falsy value of env var leads to some ambiguity. The above works more reasonably IMHO

Reply:
Works for me.